app/src/main/java/space/taran/arknavigator/ui/fragments/ResourcesFragment.kt

```diff
@@ -261,6 +252,33 @@ class ResourcesFragment(val root: Path?, val path: Path?): MvpAppCompatFragment(
         dialog = alertBuilder.show()
     }
 
+    private fun sortAccordingToChoice(){
+        when(presenter.sorting) {
+            Sorting.NAME -> gridAdapter.sortBy { it.fileName }
+            Sorting.SIZE -> gridAdapter.sortBy { Files.size(it) }
+            Sorting.TYPE -> gridAdapter.sortBy { extension(it) }
+            Sorting.LAST_MODIFIED -> gridAdapter.sortBy { Files.getLastModifiedTime(it) }
+            Sorting.DEFAULT -> {
+                sendShortToastMessage(requireActivity().getString(R.string.as_is_sorting_selected))
```

Comment:
This message about `as-is` pops up every time a user opens the resources screen, not only after making the choice itself.